presto-docs/src/main/sphinx/release/release-300.rst

```diff
@@ -0,0 +1,112 @@
+===========
+Release 300
+===========
+
+General Changes
+---------------
+
+* Fix correctness issue for :func:`array_intersect` and :func:`array_distinct`
+  when input contains both zeros and nulls.
+* Fix ``count(*)`` aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve the table scan performance for structured types.
```

Comment:
Sounds a little weird. How about either:
* Improve table scan performance for structured types.
* Improve the performance of table scans for structured types.